crates/pulsar-core/src/pdk/process_tracker.rs

```diff
@@ -53,6 +54,12 @@ struct InfoRequest {
     tx_reply: oneshot::Sender<Result<ProcessInfo, TrackerError>>,
 }
 
+struct DescendantRequest {
+    pid: Pid,
+    image: String,
+    tx_reply: oneshot::Sender<Result<bool, TrackerError>>,
```

Comment:
Since we're always sending `Ok(bool)`, we should also replace this with `oneshot::Sender<bool>` and remove the `unwrap_or(false)`